src/Compat.jl

```diff
@@ -396,6 +396,11 @@ if VERSION < v"1.5.0-DEV.124"
     export isdisjoint
 end
 
+# https://github.com/JuliaLang/julia/pull/35577
+if VERSION < v"1.5.0-DEV.675"
```

Comment:
```suggestion
if VERSION < v"1.5.0-DEV.681"
```
(At lest that's what I get from `commit-name.sh 641f444`. How did you determine that version bound?)

Reply:
oh I see,

641f444, which is the commit where the test went in
75c68d, (which I used) is the commit where the change went in

Reply:
But https://github.com/JuliaLang/julia/commit/641f444a0beec17ae93ca4af5a2992d1c31cef43 adds definition and test, no? And I can't find a 75c68d. Otherwise I'd agree with you.

Reply:
Not sure where I got confused (maybe I squashed, or force-pushed), but you're absolutely right! Sorry about that.